starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +20,43 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord (array) {
+  var longest = 0; 
+  for (var i = 0; i < array.length; i++) {
+    if (array[i].length > longest) {
+      longest = array[i].length;
```

Comment:
Os falla un pequeño detalla, en lugar de guardar la longitud del string si guardáis directamente el string y tambien tenéis el return dentro del for, si lo sacáis funcionaría.
```
var longest = array[0]; 
  for (var i = 1; i < array.length; i++) {
    if (array[i].length > longest.length) {
      longest = array[i];
    } 
  }
  return longest;
```